.github/workflows/browserstack.yml

```diff
@@ -0,0 +1,87 @@
+name: Browserstack
```

Comment:
Let's name this `BrowserStack Core 3.x` or something similar and let's rename the file as well. We will need a separate file for Core 4.x as soon as one is released and this will result in less renaming as we'll support both for at least one release with different browser sets. And even if we align browser sets when Migrate 4.0 gets released, it still makes sense to separate Core major lines to avoid one big run.